release.js

```diff
@@ -0,0 +1,50 @@
+const Promise = require("bluebird");
+const Git = require("nodegit");
+
+const bumpJSONFiles = require("./src/file/bump-json-files");
+const gitCommit = require("./src/git/git-commit");
```

Comment:
Create an `index.js` to expose these functions from the root of the module and use them here as the first client.